src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-accent-blue: #00acdc;
+$color-white: #fff;
+$color-hover-text: #34568b;
+$font-family: roboto, helvetica, sans-serif;
```

Comment:
You are setting the font-family in every possible place. Simply put it in body

Reply:
Same thing with other variables. If value of a property is inherited and you want it to be so, don't apply this value again